CMakeLists.txt

```diff
@@ -60,13 +60,22 @@ elseif(LINUX)
     endif()
 endif()
 
+if (NOT TARGET azure_macro_utils_c) 
+if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/azure-macro-utils-c/CMakeLists.txt)
```

Comment:
Tab please to keep things nice and neat.

Reply:
Fixed

---
In reply to: [306032531](https://github.com/Azure/azure-ctest/pull/33#discussion_r306032531) [](ancestors = 306032531)